tools/bpf/bpftool/Documentation/bpftool-gen.rst

```diff
@@ -215,7 +234,9 @@ This is example BPF application with two BPF programs and a mix of BPF maps
 and global variables. Source code is split across two source code files.
 
 **$ clang -target bpf -g example1.bpf.c -o example1.bpf.o**
+
```

Comment:
Those lines were missing a carriage return and all done in one line. This is a small fix together with the change. I don't think upstream would mind having it in the same commit, but if you think they're willing to ask a different commit for this then we can remove it.

Reply:
I think this is OK to have in the same commit.